### PR TITLE
Fix padding for empty _getLikeCountWidget()

### PR DESCRIPTION
**Pull request by comerc**
I don't need set likeCountPadding & countDecoration when likeCount == null

https://youtu.be/a8xkUFS5JRo

### CODE REVIEW

**Comment by zmtzawqlp** [comment]
could you please tell what issue did you meet?

**Comment by comerc** [comment]
may be this code explains: https://github.com/fluttercandies/like_button/pull/17/files#diff-85bfb4bd521f3f1a9f56caf7a91a0194

File: example/.flutter-plugins-dependencies [hunk 1]
```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"image_picker_saver","dependencies":[]},{"name":"path_provider","dependencies":[]},{"name":"url_launcher","dependencies":["url_launcher_web","url_launcher_macos"]},{"name":"url_launcher_macos","dependencies":[]},{"name":"url_launcher_web","dependencies":[]}]}
```
Comment (AlexVincent525) [inline example/.flutter-plugins-dependencies]: This file shouldn't be check into version control.

File: lib/src/like_button.dart [hunk 2]
```diff
@@ -374,6 +362,16 @@ class _LikeButtonState extends State<LikeButton> with TickerProviderStateMixin {
       clipper: LikeCountClip(),
     );
 
+    if (widget.countDecoration != null) {
```
Comment (AlexVincent525) [inline lib/src/like_button.dart]: Maybe some comments can added to here such as:
`/// No decoration and padding will be created when [countDecoration] or [likeCountPadding] is null.`

File: pubspec.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ author: zmtzawqlp <zmtzawqlp@live.com>
 homepage: https://github.com/fluttercandies/like_button
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.3.0 <3.0.0"
```
Comment (AlexVincent525) [inline pubspec.yaml]: In what reason this should be higher than 2.3.0?

**Comment by AlexVincent525** [comment]
But as I seem, there's no too much actual code have changed with those features. You can leave the properties empty in previous version and it worked.

File: pubspec.yaml (hunk 3 above)
Comment (comerc) [reply pubspec.yaml]: Yes

**Comment by comerc** [comment]
I want to use LikeButton in two cases: with likeCount and w/o likeCount.

In previous version, likeCountPadding adding spaces in both cases.

And I have dummy external code for around it:
```
likeCountPadding: widget.likeCount != null ? widget.likeCountPadding : null,
```

File: lib/src/like_button.dart (hunk 2 above)
Comment (comerc) [reply lib/src/like_button.dart]: Really? :)

File: example/.flutter-plugins-dependencies (hunk 1 above)
Comment (comerc) [reply example/.flutter-plugins-dependencies]: not found in .gitignore

**Comment by AlexVincent525** [comment]
That's not what I mean😂, you did a great job. I'll verify this tommorrow, please give me some time.😉

File: lib/src/like_button.dart (hunk 2 above)
Comment (AlexVincent525) [reply lib/src/like_button.dart]: > Really? :)

Of course, we should make this prettier.

File: lib/src/like_button.dart [hunk 4]
```diff
@@ -234,7 +222,7 @@ class _LikeButtonState extends State<LikeButton> with TickerProviderStateMixin {
           );
         },
       ),
-      likeCountWidget
+      if (_likeCount != null) _getLikeCountWidget(),
```
Comment (zmtzawqlp) [inline lib/src/like_button.dart]: not everyone has high dart sdk version

File: lib/src/like_button.dart (hunk 2 above)
Comment (zmtzawqlp) [reply lib/src/like_button.dart]: may be user doesn't want like widget but any other widget base on like count.

File: pubspec.yaml (hunk 3 above)
Comment (AlexVincent525) [reply pubspec.yaml]: Why needs higher version?

**Pull request closed without merging**